src/components/POLine/Location/LocationForm.js

```diff
@@ -23,13 +23,21 @@ const reduceLocations = (locations, propName) => {
   return locations.reduce(reducer, 0);
 };
 
+const validateNotNegative = (value) => {
+  const parsedValue = parseInt(value, 10);
+
+  return Number.isNaN(parsedValue) || parsedValue < 0
+    ? <FormattedMessage id="ui-orders.validation.quantity.canNotBeLess0" />
+    : undefined;
+};
+
 const validateQuantityPhysical = (value, { cost, locations = [] }) => {
   const allLocationsQuantity = reduceLocations(locations, 'quantityPhysical');
   const overallLineQuantity = get(cost, 'quantityPhysical', 0);
 
   return allLocationsQuantity <= overallLineQuantity
     ? undefined
-    : <FormattedMessage id="ui-orders.location.quantityPhysical.exceeds" />;
+    : <FormattedMessage id="ui-orders.location.quantityPhysical.notMatch" />;
```

Comment:
@aliaksei-chumakou somewhere mentioned that it's better to add new translation than edit existing one, as I got it correctly

Reply:
yep, but I've realized that thing like don't change only value for translation, for preventing skipping the new value with old key